plugin/src/py/android_screenshot_tests/pull_screenshots.py

```diff
@@ -250,6 +250,9 @@ def pull_screenshots(process,
 
     temp_dir = temp_dir or tempfile.mkdtemp(prefix='screenshots')
 
+    if not os.path.exists(temp_dir):
+        os.makedirs(temp_dir)
```

Comment:
If you consider the example with /build/reports/(...) it seems natural, that the directories are created by build scripts. Executing `clean` task would also mean that you have to recreate the directory each time manually.